README.md

```diff
@@ -38,6 +48,19 @@ yarn add --dev interface-forge
 pnpm add --save-dev interface-forge
 ```
 
+For Zod integration, also install Zod:
+
+```shell
+# npm
+npm install zod
+
+# yarn
+yarn add zod
+
+# pnpm
+pnpm add zod
```

Comment:
this should be declared as a peerDependency. This means it will be installed on the users machine by the package manager. No need for extra install commands.